src/main/java/net/dv8tion/jda/api/interactions/commands/CommandInteraction.java

```diff
@@ -200,4 +200,54 @@ default OptionMapping getOption(@Nonnull String name)
         List<OptionMapping> options = getOptionsByName(name);
         return options.isEmpty() ? null : options.get(0);
     }
+
+    /**
+     * Gets the slash command String for this slash command.
+     * <br>This is similar to the String you see when clicking the interaction name in the client.
+     *
+     * <p>Example return for an echo command: {@code /say echo phrase: Say this}
+     *
+     * @return The command String for this slash command
+     */
+    @Nonnull
+    default String getCommandString()
+    {
+        //Get text like the text that appears when you hover over the interaction in discord
+        StringBuilder builder = new StringBuilder();
+        builder.append("/").append(getName());
+        if (getSubcommandGroup() != null)
+            builder.append(getSubcommandGroup()).append(" ");
+        if (getSubcommandName() != null)
+            builder.append(getSubcommandName()).append(" ");
+        builder.append(" ");
```

Comment:
Unless I'm missing something, this will render out 
`"/{name}{subcommandGroup} {subcommand} "`
It should instead render out
`"/{name} {subcommandGroup} {subcommand}"`

You should update your system to do the space appending when adding the next thing, not after an item has been added.

This applies to every space appending happening in this code.

Reply:
I believe you are correct, I have pushed updated files

Reply:
implemented changes